### ISSUE TITLE
Go is downloaded multiple times

### ISSUE THREAD

**Issue by steinfurt** [cloudfoundry/nodejs-buildpack#674]
What version of Cloud Foundry and CF CLI are you using? (i.e. What is the output of running `cf curl /v2/info && cf version`?
- Not relevant, I think.

What version of the buildpack you are using?
- The latest currently available one, v1.8.19.

If you were attempting to accomplish a task, what was it you were attempting to do?
- Deploying a Node.js based app with this buildpack (`cf push`). But it also happens for other apps/buildpacks.

What did you expect to happen?
- Go should only be downloaded once.

What was the actual behavior?
- Go is actually downloaded twice, once during the [supply step](https://github.com/cloudfoundry/nodejs-buildpack/blob/v1.8.19/bin/supply#L10), once during the [finalize step](https://github.com/cloudfoundry/nodejs-buildpack/blob/v1.8.19/bin/finalize#L11). The reason for this is that there is a small error in the [line 21](https://github.com/cloudfoundry/nodejs-buildpack/blob/v1.8.19/scripts/install_go.sh#L21): Instead of `[[ ! -f "${dir}/go/bin/go" ]]` it should be `[[ ! -f "${dir}/bin/go" ]]`.

Can you provide a sample app?
- There is no need to provide a sample app, I think. But the issue can be easily reproduced with the following terminal commands:

```
cd scripts
export CF_STACK=cflinuxfs3
./install_go.sh
./install_go.sh
```

A log output with `-----> Download go 1.19` appearing twice can be seen in #650.

The issue makes the errors reported in #626 and #650 more likely to appear. Of course, I could provide a PR, but since it relates to a script shared in [many other buildpacks](https://github.com/search?q=org%3Acloudfoundry+%22%24%7Bdir%7D%2Fgo%2Fbin%2Fgo%22&type=code), I think this should probably be handled centrally.

Please confirm where necessary:
* [X] I have included a log output
* [ ] My log includes an error message
* [X] I have included steps for reproduction

**Comment by brayanhenao** [comment]
Thank you for bringing this issue to our attention, @steinfurt. We are addressing it by implementing a fix in the script. You can track the progress and changes in the following pull request: https://github.com/cloudfoundry/buildpacks-github-config/pull/65. This update will ensure that the necessary changes are propagated across all Buildpacks.

**Comment by steinfurt** [comment]
Thank you for your quick reply, @brayanhenao! Now I also know where the scripts are handled centrally... ;-)

The fix has been released as part of v1.8.21.
